compliance/utils/fetch_command

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+# Copyright (c) 2021 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+usage() {
+  echo "Usage: $0 [-h] [-u <user>]"
+  echo "       -h    Print usage."
+  echo "       -s    Show exit status."
+  return 1
+} 1>&2
+
+show_exit_status=0
+while getopts :hs option
+do
+  case $option in
+    h)
+      usage
+      exit 0
+      ;;
+    s)
+      show_exit_status=1
+      ;;
+    *)
+      echo "Error: Illegal option -$OPTARG specified." 1>&2
+      exit "$(usage)"
+      ;;
+  esac
+done
+shift $((OPTIND - 1))
+
+ps1() {
+  printf '%s@%s:%s$' "$(whoami)" "$(hostname)" "${PWD/#$HOME/~}"
```

Comment:
Yup, +1 to `hostname -f`. I'm unsure about adding a timestamp; it would alter the digest on every fetch. Users can put `date -u +%FT%TZ` in their fetcher code if it's really required.
The signature is always different on every encrypt regardless of the content. I had planned to compare digests and not upload new evidence if there's a matching digest already in the locker. Although this isn't included in this PR.

Reply:
Yeah, it'd be nice to not commit anything to Git if the command output is the same. But I think the timestamp is quite important to auditors, and you want it signed as well. It's immediately clear to anyone reading the code that the hostname and timestamp were taken from the server and then signed with its key.

Reply:
Okay, I'll add it in if preferred. For reference, here's what I had planed as an example fetcher. Note the full hostname and timestamp are already included as commands to be executed. I guess this would no longer be required.
```python
@store_raw_evidence('encryption/srv_crypt.txt')
def fetch_encryption(self):
    return self.fetchCommand([
        'date --utc',
        'hostname -f',
        'mount | grep srv',
        'cryptsetup status srv'
    ])
```

Reply:
I've add an additional kwarg to support this. The above snippet would become:
```py
@store_raw_evidence('disk/df.txt')
def fetch_disk(self):
    return self.fetchLocalCommands([
        'df -h'
    ], show_exit_status=True, show_timestamp=True)
```
```
[2022-01-21T11:54:56Z] samsmith@Sams-IBM-MacBook-Pro.local:~$ df -h 
Filesystem      Size  Used Avail Use% Mounted on
/dev/disk1s5s1  466G  181G  286G  39% /
[2022-01-21T11:54:56Z] samsmith@Sams-IBM-MacBook-Pro.local:~$ echo $?
0
```

Reply:
Fine for it to be options here. I think we just run the risk of having some evidence timestamped and some not, depending on whether people think to enable, whereas auditors should always need it.